src/cpp/FilesInformation/Loaders/FilesByNameLoader.cpp

```diff
@@ -77,19 +77,19 @@ TEST_CASE("Extension with several dots with Windows separator") {
 
 TEST_CASE("Existing file") {
     FilesByNameLoader loader = FilesByNameLoader();
-    File file = loader.loadFile("resources/text.txt");
+    File file = loader.loadFile("../../../../resources/text.txt");
```

Comment:
This path should be relative to executable location, not source file location. It probably won't work on all IDEs